app/decode/common.js

```diff
@@ -32,3 +32,12 @@ export function permute(permutation) {
   }
   return result
 }
+
+export function alternativePermutations(listToPermutate) {
+  const originalPostions = [...listToPermutate]
+  let allPermutations = permute(listToPermutate)
```

Comment:
Add comment to `permute` method that original permutation is always the first one.